website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -4,391 +4,275 @@ linkTitle: Releasing
-
-Unlock `master` or release branch after the release has completed (after step 6 is complete) by changing the number of required approving reviewers back to 1.
-
-### Releasing
-
-The GitHub [`Releases` tab][release-page] in the operator-sdk repo is where all SDK releases live.
-To create a GitHub release see the [releasing binaries section](#9-releasing-binaries-signatures-and-release-notes).
+### 2. Lock down branch `master`
 
-#### Release notes
-
-GitHub release notes should thoroughly describe changes made to code, documentation, and design of the SDK. PR links should be included wherever possible.
-
-The following sections, often directly copied from our [changelog][doc-changelog], are used as release notes:
-
-```Markdown
-[Version as title, ex. v1.2.3]
-
-### Added
-- [Short description of feature added] (#PR)
-...
-
-### Changed
-- [Short description of change made] (#PR)
-...
-
-### Deprecated
-- [Short description of feature deprecated] (#PR)
-...
+Lock down the `master` branch to prevent further commits before the release completes.
+See [this section](#locking-down-branches) for steps to do so.
 
-### Removed
-- [Short description of feature removed] (#PR)
-...
+### 3. Create and merge a new PR
 
-### Bug Fixes
-- [Short description of bug and fix] (#PR)
-...
-```
+Create and merge a new PR for the above commit. You can force-merge your PR to the locked-down `master`
+if you have admin access to the operator-sdk repo, or ask an administrator to do so.
 
-## Release Signing
+### 4. Unlock the `master` branch
 
-When a new release is created, the tag for the commit it signed with a maintainers' gpg key and
-the binaries for the release are also signed by the same key. All keys used by maintainers will
-be available via public PGP keyservers such as pool.sks-keyservers.net.
+See [this guide](#unlocking-branches) for steps to do so.
 
-For new maintainers who have not done a release and do not have their PGP key on a public
-keyserver, output your armored public key using this command:
+### 5. Create and push a release tag
 
 ```sh
-$ gpg --armor --export "$GPG_EMAIL" > mykey.asc
+make tag RELEASE_VERSION=v1.3.0
+git push upstream v1.3.0
 ```
 
-Then, copy and paste the content of the outputted file into the `Submit a key` section on
-pool.sks-keyservers.net or any other public keyserver that synchronizes
-the key to other public keyservers. Once that is done, other people can download your public
-key and you are ready to sign releases.
-
-## Verifying a release
+### 6. Fast-forward the `latest` and release branches
 
-To verify a git tag, use this command:
+The `latest` branch points to the latest release tag to keep the main website subdomain up-to-date.
+Run the following commands to do so:
 
 ```sh
-$ git verify-tag --verbose "$TAG_NAME"
+git checkout latest
+git reset --hard tags/v1.3.0
+git push -f upstream latest
```

Comment:
Can this be in the Makefile too? `make update-latest`?

Reply:
Will add in a follow-up.